cmd/govim/testdata/signs.txt

```diff
@@ -8,6 +8,9 @@
 [vim] [!vim:v8.1.1682] skip
 [gvim] [!gvim:v8.1.1682] skip
 
+# Ensure sign placment are enabled
+vim call 'govim#config#Set' '["QuickfixSignsDisable", 0]'
```

Comment:
Isn't the intention here to verify the default is disabled instead?

In which case, perhaps this line shifts to another testscript where we assert the value returned by `govim#config#Get()`? i.e. we create a testscript for asserting the out-of-the-box config for all settings? Thoughts?

Reply:
Yeah, good point.
I thought about adding `govim#config#Get()` to `quickfix_config.txt`, but it isn't populated with default values if there are none (maybe it should?) so I just tested the default behaviour there.

I also had an idea of maybe splitting up the `quickfix_config.txt` into different versions since the version handling clutters the readability. Then it would be easy to drop, for example, `quickfix_config_pre_v8.1.1682` if that version isn't supported anymore. But I decided to leave it as is for now.

I'll remove the call from `signs.txt` as solution for this .